src/App.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import './App.scss';
+
+import Clock from './components/Clock';
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  };
+
+  showClock = () => {
+    this.setState({ isClockVisible: true });
+  };
+
+  hideClock = () => {
+    this.setState({ isClockVisible: false });
+  };
+
+  randomName = () => {
```

Comment:
```suggestion
  makeRandomName = () => {
```
better to name the methods according to what they actually do